app/templates/components/notification-dropdown.hbs

```diff
@@ -0,0 +1,48 @@
+<i class="large mail outline icon">
```

Comment:
@niranjan94 as the stylings are for this component only. Would it be better if I put all the styling in a single file under styles/components  instead of in multiple files as it is right now ?